airflow/www/templates/airflow/graph.html

```diff
@@ -43,37 +51,36 @@
     <input name="_csrf_token" type="hidden" value="{{ csrf_token() }}">
     <input type="submit" value="Go" class="btn btn-default">
   </form>
-  <div class="input-group" style="float: right;">
+  <div class="input-group pull-right">
     <input type="text" id="searchbox" class="form-control" placeholder="Search for...">
   </div>
-  <div style="clear: both;"></div>
 </div>
 <hr/>
-<div>
-  {% for op in operators %}
-  <div class="legend_item"
-       style="border-width:1px;float:left;background:{{ op.ui_color }};color:{{ op.ui_fgcolor }};">
-    {{ op.__name__ }}
+<div class="clearfix">
+  <div class="legend legend--pills pull-left">
+    {% for op in operators %}
+      <div class="legend__item"
+           data-operator-name="{{ op.__name__ }}">
+        {{ op.__name__ }}
+      </div>
+    {% endfor %}
   </div>
-  {% endfor %}
-
-  <div>
-    <div class="legend_item state" style="border-color:white;">no_status</div>
-    <div class="legend_item state" style="border-color:grey;">queued</div>
-    <div class="legend_item state" style="border-color:gold;">up_for_retry</div>
-    <div class="legend_item state" style="border-color:turquoise;">up_for_reschedule</div>
-    <div class="legend_item state" style="border-color:pink;">skipped</div>
-    <div class="legend_item state" style="border-color:red;">failed</div>
-    <div class="legend_item state" style="border-color:lime;">running</div>
-    <div class="legend_item state" style="border-color:green;">success</div>
+  <div class="legend legend--pills legend--state pull-right">
```

Comment:
I don't think we need legend and legend--pills - it seems the later is only used in JS to select, so we could use legend for that.

Reply:
It is also used in the CSS code.
https://github.com/apache/airflow/pull/4823/files#diff-eddd096d3961f002a677cf7d45bd1657R60
https://github.com/apache/airflow/pull/4823/files#diff-eddd096d3961f002a677cf7d45bd1657R27

Reply:
is `.legend` used though?

Reply:
Currently no, but I think it's useful because it makes it easier to create themes. This allows you to grab all elements of one type with one selector.

Reply:
Do you think I should delete it?

Reply:
I think it will save us confusion in the future if we don't have extra classes that aren't  used by CS/JSS

Reply:
Ok. I will delete it in a moment.